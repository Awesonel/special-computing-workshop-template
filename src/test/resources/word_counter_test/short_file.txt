foo foo bar
bar foo kar
FOO KAR bar