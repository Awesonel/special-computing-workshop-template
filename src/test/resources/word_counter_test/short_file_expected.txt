bar : 3
foo : 4
kar : 2
